src/simulator/services/resources/directory.py

```diff
@@ -67,7 +67,8 @@ def load(self, name: str, load_function: Callable[['Directory', str], Any] = Non
             self._services.debug.write("Loaded [{}]".format(self._full_path() + name), DebugLevel.LOW)
         else:
             self._services.debug.write("File not found [{}]".format(self._full_path() + name), DebugLevel.LOW)
-            raise RuntimeError("File not found, consider running from src/")
+            print(self._full_path() + name)
```

Comment:
take this print statement out and ideally restore the exception. If you're debugging set the debug level high to get output

Reply:
Consider adding a debug flag to `main.py` with optionally specified level. If level not specified goes to highest possible.